### ISSUE TITLE
Update to 4.51.1 error and fix

### ISSUE THREAD

**Issue by mkz212** [oznu/homebridge-config-ui-x#1606]
### Describe The Bug

The system does not start after updating to 4.51.1.

Solution:
Connect using SSH and execute the command:
`sudo hb-service rebuild`

### Homebridge UI Version

4.51.1

### Homebridge Version

1.6.1

### Node.js Version

18.18.2

### Operating System

Official Homebridge Raspberry Pi Image

### Environment Info

- [ ] Using Docker?
- [ ] Using Hyper-V?
- [ ] Using hb-service?

### Raspberry Pi Model

Raspberry Pi 4 B

**Comment by tpitale** [comment]
Experiencing the same thing on Synology with node 18.15, and UI version 4.51.2. Except, `sudo hb-service rebuild` gives and error about not being able to find `make` 🤔

EDIT: this thread https://github.com/homebridge/homebridge-syno-spk/issues/138 has a lot more info. I'm missing compiling tools: https://github.com/homebridge/homebridge-syno-spk/wiki/DSM-7:-Enable-Compiling-Of-Native-Modules but probably not going to install that (didn't need it before …?)

**Comment by mkz212** [comment]
@tpitale This is a security, so that everyone who should not update the node (for lack of the appropriate GLIBCXX) does not update the node because homebridge will not start. 

The only way out to have node 20 is a fresh installation and restoring the backup. But wait - a new Homebridge Image based on the new Linux Bookworm will soon come out.

**Comment by donavanbecker** [comment]
This is a notice on the latest Homebridge UI release

**Comment by mkz212** [comment]
@tpitale 

Such errors happen on older installations, especially if you have a version based on older Linux (the current one is Bullseye, and the older one is e.g. Buster).

The next error is when updating the node.

The best way out is a fresh installation and restoring the backup. But new Homebridge Image based on the new Linux Bookworm will soon come out - so if you can then wait for it.

**Comment by NorthernMan54** [comment]
We have released Homebridge-config-ui-x-4.52.0-beta.33 which should resolve the issue where people are having issues running `sudo hb-service-rebuild` after updating homebridge-config-ui-x on linux based environments and needed to rollback to the previous version.   We are looking for testers that want to give it a try.  So far we have had one positive test result.

![image](https://github.com/homebridge/homebridge-config-ui-x/assets/19808920/216a4a7e-8e5b-48b5-8e12-2f89c578458e)

**Comment by mkz212** [comment]
@NorthernMan54 For me, the last update to 4.51.2 went flawlessly. It seems to me that it may also depend on whether you have Linux Buster or Bullseye - when I had Buster I had a problems with the updates and on Bullseye I did not.

**Comment by donavanbecker** [comment]
Are you having the same issues with 4.52.1?

**Comment by mkz212** [comment]
@donavanbecker 

From what I remember, I had no problem with 4.52.0. Now I have updated to 4.52.1 without a problem. 

But I emphasize that I had problems on the old installation (Homebridge Image v 1.0.21 where was Linux Buster) and then I made a fresh installation on 1.0.41 where is Linux Bullseye. Therefore, I am not sure if this error depends on the Homebridge Config Ui version or the Homebridge Image version.
